test/test_transformers.py

```diff
@@ -87,37 +87,85 @@ def use_deterministic_algorithims(mode: bool, warn_only: bool):
 isSM5xDevice = torch.cuda.is_available() and torch.cuda.get_device_capability()[0] == 5
 isLessThanSM80Device = torch.cuda.is_available() and torch.cuda.get_device_capability()[0] < 8
 
-def get_rtol(true_value: torch.Tensor, computed_value: torch.Tensor) -> float:
-    deviation = true_value - computed_value
-    deviation = torch.abs(deviation / true_value)
-    # Fill in the nans with the default rtol
-    torch.nan_to_num_(deviation, nan=default_rtol[computed_value.dtype])
-    return deviation.max().item()
 
+def _check_equal(
+    golden: torch.Tensor,
+    reference: torch.Tensor,
+    test: torch.Tensor,
+    fudge_factor: float,
+    tensor_name: Optional[str] = None
+) -> None:
+    """
+    Compare test tensor against golden and reference tensors.
+    Raises ValueError if compiled error exceeds threshold.
+    """
+    if golden.is_nested and reference.is_nested and test.is_nested:
+        for gold, ref, tst in zip(golden.unbind(), reference.unbind(), test.unbind()):
+            _check_equal(gold, ref, tst, fudge_factor, tensor_name)
+        return
```

Comment:
will this become unnecessary when NJTs support max? or will NSTs be sent through here

Reply:
hmm I think this would not be necessary for NJT but the tests in this file only run on NST so will still likely be needed